.github/workflows/pr-checks.yml

```diff
@@ -45,34 +45,37 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: '22'
+          node-version: 22
 
       - name: Setup pnpm
-        uses: pnpm/action-setup@v3
+        uses: pnpm/action-setup@v2
```

Comment:
why downgrade to v2? it seems v4 is latest?

Reply:
There was some weird issue, apparently with the pnpm cache, I was trying everything to fix, still not sure what actually worked.